src/freenet/node/RequestSender.java

```diff
@@ -1716,9 +1722,7 @@ private boolean finishOpennet(PeerNode next) {
 				// RequestHandler will send a noderef back up, eventually, and will unlockHandler() after that point.
 				// But if this is a local request, we need to send the ack now.
```

Comment:
is this comment still up to date with this change, or should the “local request needs ack” line  be removed?